Makefile

```diff
@@ -240,8 +240,10 @@ front_build:
 
 
 checks: front_build front_lint check_perltidy check_perl_fast check_critic
+# TODO: add check_taxonomies when taxonomies ready
 
 lint: lint_perltidy
+# TODO: add lint_taxonomies when taxonomies ready
```

Comment:
Is it dependent on this: https://github.com/openfoodfacts/openfoodfacts-server/pull/9843 ?

Reply:
Not really, but the other one is now merged.

It's more when taxonomy will be sorted !